v0.17.1.7-freebsd/hyc/monero-freebsd-0.17-build.assert

```diff
@@ -0,0 +1,105 @@
+--- !!omap
+- out_manifest: '50a36a796cbe3de569c26344af311b43afb0a44693383c088685830876a0f0e1  monero-x86_64-unknown-freebsd-v0.17.1.7.tar.bz2
+
+    '
```

Comment:
Ah, seems you are getting the same error I have been seeing...
```
ERROR: Found conflicting contents on line: 3 of file v0.17.1.7-freebsd/iDunk/monero-freebsd-0.17-build.assert:
'
v0.17.1.7-freebsd/hyc/monero-freebsd-0.17-build.assert:
    '
```
There is 4 leading spaces before an apostrophe in your file.

For some reason my builds also have these for `freebsd` and `osx`, so I have been manually removing the spaces assuming there was just something wonky with my build env.
PS: I'm using ubuntu 20.0.4 host.